lib/std/fmt/parse_hex_float.zig

```diff
@@ -341,7 +341,6 @@ test "f128" {
         // // Min denormalized value.
         .{ .s = "0x1p-16494", .v = math.f128_true_min },
         .{ .s = "-0x1p-16494", .v = -math.f128_true_min },
-        .{ .s = "0x1.edcb34a235253948765432134674fp-1", .v = 0x1.edcb34a235253948765432134674fp-1 },
```

Comment:
I wouldn't consider this a duplicate. This case is not covered by the tests in this file. The test for the self-hosted compiler does not count; it's testing something different and it's not part of the standard library tests.

Reply:
Reverted in db4c4cfb